pyanaconda/threading.py

```diff
@@ -303,8 +306,7 @@ def initThreading():
        an event is triggered in the GTK main loop. And IT HAS TO BE CALLED IN
        THE MAIN THREAD.
     """
-    global threadMgr
-    threadMgr = ThreadManager()
+    threadMgr.set_current_thread_as_main()
 
 
-threadMgr = None
+threadMgr = ThreadManager()
```

Comment:
Yeah, this is basically what I had in mind, except I want to get rid off `initThreading()` method and `threadMgr`.